package.json

```diff
@@ -42,13 +43,10 @@
     "eslint-plugin-react": "^7.28.0",
     "eslint-plugin-react-hooks": "^4.3.0",
     "husky": "^8.0.1",
-    "jest": "^26.6.3",
+    "jest": "^28.1.3",
     "metro-react-native-babel-preset": "^0.66.2",
     "prettier": "^2.7.1",
-    "react-test-renderer": "17.0.2",
+    "react-test-renderer": "18.0.0",
```

Comment:
This fixed the tests running, previously they'd run and fail with a 'can't read undefined object property current'